pkgs/development/python-modules/geometric/default.nix

```diff
@@ -0,0 +1,35 @@
+{ buildPythonPackage, lib, fetchPypi
+, networkx, numpy, scipy, six
+, pytest-cov, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "geometric";
+  version = "0.9.7.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1p9vxvzvnb6nnqlxl2r75jrih9bbqqbsm5zjn3i311qq3b1whsbg";
+  };
+
+  propagatedBuildInputs = [
+    networkx
+    numpy
+    scipy
+    six
+    pytest-cov
+  ];
+
+  checkInputs = [
```

Comment:
I think the prefered version is to add the `pytestCheckHook`.

Reply:
Using `pytestCheckHook` does not seem to work at all. I just get test failures. (maybe I am using it wrong?).

Reply:
Yup, just tried it myself. `pytest` and `pytest-cov` simply don't execute any tests (`0 tests ran in 0.000s`) as it seems not to discover them, while `pytestCheckHook` runs them and fails due to missing .xyz files and interfaces.

Reply:
For example geometric tries to access `geometric/data/water5.xyz`, which does actually not exist. Comparing the tarball from PyPi https://pypi.org/project/geometric/#files and Geometric's GitHub https://github.com/leeping/geomeTRIC/releases/tag/0.9.7.2 actually shows that some files are missing.  With the GitHub sources it actually works.

```
  src = fetchFromGitHub {
    owner = "leeping";
    repo = "geomeTRIC";
    rev = version;
    hash = "sha256-QFpfY6tWqcda6AJT17YBEuwu/4DYPbIMJU1c9/gHjaA=";
  };

  checkInputs = [ pytestCheckHook ];
```

Reply:
Thanks for diving into it. I changed it to GitHub sources and pytestCheckHook.